index.js

```diff
@@ -19,8 +19,29 @@ function fastifyNext (fastify, options, next) {
     .then(() => {
       fastify
         .decorate('next', route.bind(fastify))
+        .decorateReply('nextRender', null)
+        .addHook('onRequest', (request, reply, done) => {
+          reply.nextRender = path => {
+            assert(typeof path === 'string', 'path must be a string')
+
+            for (const [headerName, headerValue] of Object.entries(reply.getHeaders())) {
+              if (headerName === 'content-length' && headerValue === undefined) {
+                continue
+              }
+
+              reply.raw.setHeader(headerName, headerValue)
```

Comment:
Can you please add a comment to why this is needed.

Reply:
It's https://github.com/fastify/fastify-nextjs/pull/112, but I can add a comment probably 😀

Reply:
done